.github/workflows/chroma-integration-test.yml

```diff
@@ -22,7 +22,8 @@ jobs:
                    "chromadb/test/property/test_cross_version_persist.py",
                    "chromadb/test/property/test_embeddings.py",
                    "chromadb/test/property/test_filtering.py",
-                   "chromadb/test/property/test_persist.py"]
+                   "chromadb/test/property/test_persist.py",
+                   "chromadb/test/utils/test_embedding_functions.py"]
```

Comment:
We shouldn't need to add this here as it should be covered by the --ignore-glob 'chromadb/test/property/*' matrix entry

Reply:
If some tests change the state of our environment, should we create a new job with a new matrix, or we can use the same job and just change the matrix ?